agents/plugins/smart

```diff
@@ -108,6 +108,20 @@ if which smartctl > /dev/null 2>&1 ; then
             continue
         fi
 
+        # Exclude removable media
+        if which udevadm > /dev/null 2>&1; then
+            TMP_FILE=$(mktemp)
+            udevadm info /dev/$N > $TMP_FILE
+            if grep -Fxq "E: ID_CDROM_CD=1" $TMP_FILE; then
```

Comment:
Does this also include DVD drives?

Reply:
It basically includes everything, that is able to read a CD.
But I'm not sure if it also includes Blu-Ray drives.

I attached the whole [output of udevadm info](https://github.com/tribe29/checkmk/files/6405552/udevadm.txt). there would be a lot more to match on:
- `S: dvdrw`
- `S: cdrw`
- `S: dvd`
- `S: cdrom`
- `E: ID_TYPE=cd`

Reply:
Ok, I think this approach should work in principle